core/app/services/spree/import_service/products/create.rb

```
@@ -0,0 +1,54 @@
+        def initialize(row:)
+          @row = row
+        end
+
+        def call
+          create_product
+          built_variant = Spree::Variant.new(variant_attributes)
+          Spree::ImportService::Products::Serializers::OptionValuesAttributesSerializer.new(row: row, variant_id: nil).to_a.map do |attributes|
+            built_variant.option_value_variants.build.assign_attributes(attributes)
+          end
+          built_variant.save!
+          
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)